Source/CompSpinalEnginePowerTrader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using RimWorld;
+using Verse;
+
+namespace TheCafFiend
+{
+    [StaticConstructorOnStartup]
+    public class CompSpinalEnginePowerTrader : CompPowerTrader
+    {
+        public override void SetUpPowerVars()
+        {
+            base.SetUpPowerVars();
+            CompSpinalEngineTrail engineToModify = parent.TryGetComp<CompSpinalEngineTrail>();
+            if (engineToModify == null) // Like this.... Probably? can't happen but just in case
```

Comment:
🤔  Ok I learned about the existence of a bunch of things reading what this even is, hah. Would... That not enable me to log a message if it was null though? Rimworld mods, due to the world of everyone using harmony injection (aka reflection, right?) are infamous for spewing nullrefs due to mod conflicts because some monster transpiled your base type, so I want some sort of an indication. Is that still an option?